dev-tools/packer/platforms/centos/init.j2

```diff
@@ -42,8 +44,12 @@ if status | grep -q -- '-p' 2>/dev/null; then
     pidopts="-p $pidfile"
 fi
 
+if [ -f /sbin/runuser ]; then
```

Comment:
Rather than explicitly stating the path, I think it would be better to use the check described [here](http://stackoverflow.com/a/4785518/503798). WDYT?